tests/fixtures/brand-feed.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Feed xmlns="http://www.bazaarvoice.com/xs/PRR/ProductFeed/5.6" name="BrandFeed" incremental="false" extractDate="2018-07-12T19:22:32"><Brands><Brand><ExternalId>first-brand</ExternalId><Name>First Brand</Name></Brand><Brand><ExternalId>second-brand</ExternalId><Name>Second Brand</Name></Brand></Brands></Feed>
```

Comment:
formatting this fixtures makes easier to read

Reply:
The problem here is that we generate a xml like this. So if we indent this file, the test fails. I will try a better assertion for xml and see if we can do this here.

Reply:
I guess xml assertions from PHPUnit handle that

Reply:
already fixed... nervermind